doc/rados/configuration/common.rst

```diff
@@ -100,75 +104,78 @@ Additionally, you should enable message signing. See `Cephx Config Reference`_ f
 OSDs
 ====
 
-Ceph production clusters typically deploy :term:`Ceph OSD Daemons` where one node
-has one OSD daemon running a Filestore on one storage device. The BlueStore back
-end is now default, but when using Filestore you specify a journal size. For example:
+When Ceph production clusters deploy :term:`Ceph OSD Daemons`, the typical
+arrangement is that one node has one OSD daemon running Filestore on one
+storage device. BlueStore is now the default back end, but when using Filestore
+you must specify a journal size. For example:
 
 .. code-block:: ini
 
-	[osd]
-	osd_journal_size = 10000
+    [osd]
+    osd_journal_size = 10000
 
-	[osd.0]
-	host = {hostname} #manual deployments only.
+    [osd.0]
+    host = {hostname} #manual deployments only.
 
 
-By default, Ceph expects to store a Ceph OSD Daemon's data at the
-following path::
+By default, Ceph expects to store a Ceph OSD Daemon's data on the following
+path::
 
-	/var/lib/ceph/osd/$cluster-$id
+    /var/lib/ceph/osd/$cluster-$id
 
-You or a deployment tool (e.g., ``cephadm``) must create the corresponding
-directory. With metavariables fully expressed and a cluster named "ceph", this
-example would evaluate to::
+You or a deployment tool (for example, ``cephadm``) must create the
+corresponding directory. With metavariables fully expressed and a cluster named
+"ceph", the path specified in the above example evaluates to::
 
-	/var/lib/ceph/osd/ceph-0
+    /var/lib/ceph/osd/ceph-0
 
-You may override this path using the ``osd_data`` setting. We recommend not
-changing the default location. Create the default directory on your OSD host.
+You can override this path using the ``osd_data`` setting. We recommend that
+you do not change the default location. To create the default directory on your
+OSD host, run the following commands:
 
 .. prompt:: bash $
 
-	ssh {osd-host}
-	sudo mkdir /var/lib/ceph/osd/ceph-{osd-number}
+    ssh {osd-host}
+    sudo mkdir /var/lib/ceph/osd/ceph-{osd-number}
 
-The ``osd_data`` path ideally leads to a mount point with a device that is
-separate from the device that contains the operating system and
-daemons. If an OSD is to use a device other than the OS device, prepare it for
-use with Ceph, and mount it to the directory you just created
+The ``osd_data`` path ought to lead to a mount point that has mounted on it a
```

Comment:
s/ought to/should/
But really this whole section is a throwback to Filestore.  I honestly would just remove at least lines 141-155.  They might still apply to deployments that use ``dmcrypt``, but my Quincy Rook deployments have the OSD metadata directories right on `/var/lib` and I think `cephadm` / `ceph-volume` deployments do as well.  Moreover, with Rook and I think with `ceph-volume` the OSD metadata directories are no longer named for the OSD ID:

```
[root@ceph21 ~]# ls /var/lib/ceph/rook-ceph/
90562b22-d5e9-45b8-94a2-7d33217a9a01_02cb5520-6db2-4ec8-9daa-a1108ca65ec8  90562b22-d5e9-45b8-94a2-7d33217a9a01_5a7b3f31-754f-4081-81e2-9c8a401a6fdc  90562b22-d5e9-45b8-94a2-7d33217a9a01_c97e91c3-5416-48f2-85c4-f4fb76c5a4b0
90562b22-d5e9-45b8-94a2-7d33217a9a01_1a0969c8-cc34-43e5-b621-96c38d6010fa  90562b22-d5e9-45b8-94a2-7d33217a9a01_613bb6c3-53c7-41fd-8fe1-de890af3a5ff  90562b22-d5e9-45b8-94a2-7d33217a9a01_d9e2140b-151b-4e0b-832a-ace01d4d4f05
90562b22-d5e9-45b8-94a2-7d33217a9a01_1a993fda-7cac-4950-ad13-bc9323f50a1b  90562b22-d5e9-45b8-94a2-7d33217a9a01_70500946-45cd-4502-8c98-3be3f608b9c0  client.admin.keyring
90562b22-d5e9-45b8-94a2-7d33217a9a01_24354b59-6889-4992-98c4-c2eea905eeb3  90562b22-d5e9-45b8-94a2-7d33217a9a01_838843dd-247b-44f8-a67b-3efadbf7d111  crash
90562b22-d5e9-45b8-94a2-7d33217a9a01_3effa610-d837-46f9-b2c7-f5bd92e981fb  90562b22-d5e9-45b8-94a2-7d33217a9a01_af980f69-0f3c-436d-b2b2-7d1d48a94964  log
90562b22-d5e9-45b8-94a2-7d33217a9a01_3f020ed9-2d0a-4775-9d41-a08529377b0c  90562b22-d5e9-45b8-94a2-7d33217a9a01_b488531a-a82c-401e-bf82-a5fc9fe289fe  rook-ceph.config
90562b22-d5e9-45b8-94a2-7d33217a9a01_4e545e04-c633-4682-aeb0-4da2ef175e4c  90562b22-d5e9-45b8-94a2-7d33217a9a01_c119d558-baf1-4471-85d7-61389c902fe2
..
[root@ceph21 rook-ceph]# cat 90562b22-d5e9-45b8-94a2-7d33217a9a01_838843dd-247b-44f8-a67b-3efadbf7d111/whoami
124
```

Reply:
unfinished business